### DISCUSSION TITLE
Where to execute "infracost auth login"

### DISCUSSION THREAD

**Discussion by tacodder** [infracost/infracost#2444] [Q&A]
Tutorial (https://www.infracost.io/docs/#2-get-api-key ) to execute Infracost in Docker is saying that you need to execute "infracost auth login" but I don't know where to execute if in my OS Windows or inside the container, also the container doesn't execute wtihout it. Any one can help me?

docker pull infracost/infracost:ci-latest

docker run --rm \
  -e INFRACOST_API_KEY=see_following_step_on_how_to_get_this \
  -v $PWD/:/code/ infracost/infracost:ci-latest breakdown --path /code/
  # Add other required flags/envs, e.g. --terraform-var-file or --terraform-var

**Comment by alikhajeh1** [comment]
@tacodder please execute that outside of the container and pass-in the API key using the env var. You can also simply go to https://www.infracost.io/ and login then get the API key from the Org Settings page - that's what the majority of docker users do.
